README.md

```diff
@@ -16,6 +16,29 @@ cd segmentmytiff
 python -m pip install .
 ```
 
+## Train a feature extraction model
+
+To train a feature extraction model run the script "train_model.py" in this repo:
+```bash
+python ./src/segmentmytiff/utils/train_model.py -r ../monochrome_flair_1_toy_dataset_flat/ --train_set_limit 10
```

Comment:
Maybe is worth to have a folder `scripts` that calls `train_model.py`.
Also, may be useful to have an (empty) folder `data` where you can put files like `monochrome_flair_1_toy_dataset_flat` that is the data to train the model and lives only locally, so not committed to the repo/
What do you think?